aule/visitors/alfaASTVisitor.py

```diff
@@ -13,14 +13,10 @@ def visitEffectStatement(self, ctx):
     def visitRulelist(self, ctx):
         return self.visitChildren(ctx)
 
-    def visitCallExpression(self, ctx):
-        callee = ctx.ID().getText()
-        return CallExpression(callee)
-
     def visitAttributeAccessExpression(self, ctx):
         if not ctx.attributeAccessExpression():
             return Identifier(ctx.ID().getText())
-        return AttributeAccessExpression(self.visit(ctx.attributeAccessExpression()), ctx.ID().getText() )
+        return AttributeAccessExpression(self.visit(ctx.attributeAccessExpression()), ctx.ID().getText())
 
     def visitAttributeValue(self, ctx):
         text = ctx.getText()
```

Comment:
Below we have a comparison `if text.lower() == "true" or text.lower() == "false":` and then we create `LiteralBoolean(text)`

So we can have `LiteralBoolean("TrUe")` and others which seems not very convenient to use.

Need to change constructor call to lowercase also

Reply:
Done

Reply:
Still there https://github.com/PositiveTechnologies/aule/blob/update-alfa-grammar/aule/visitors/alfaASTVisitor.py#L24